CommonTools/ParticleFlow/python/EITopPAG_cff.py

```diff
@@ -26,11 +26,11 @@
 #### condenses information into the new "particleFlow" collection.            ####
 
 
-pfPileUpEI = pfPileUp.clone( PFCandidates = cms.InputTag('particleFlowPtrs') )
+pfPileUpEI = pfPileUp.clone( PFCandidates = cms.InputTag('particleFlowPtrs'), Jets=cms.InputTag('ak4PFJets') )
 pfNoPileUpEI = pfNoPileUp.clone( bottomCollection = cms.InputTag('particleFlowPtrs'),
                                  topCollection = cms.InputTag('pfPileUpEI') )
 
-pfPileUpJMEEI = pfPileUpJME.clone( PFCandidates = cms.InputTag('particleFlowPtrs') )
+pfPileUpJMEEI = pfPileUpJME.clone( PFCandidates = cms.InputTag('particleFlowPtrs'), Jets=cms.InputTag('ak4PFJets') )
```

Comment:
True, but this has no effect on the defaults and simplifies the toModify statements.
I can move things there if preferred.

I don't expect significant changes to implementation unless there are strong feelings about @arizzi comment about calling PVAssociator class directly from PFPileup.